monkey/monkey_island/cc/models/zero_trust/test_segmentation_finding.py

```diff
@@ -1,56 +0,0 @@
-import pytest
```

Comment:
What about all these tests?

Reply:
Segmentation finding no longer exists. Other tests were fixed and added some new ones